tests/schema/test_library.py

```diff
@@ -36,12 +36,6 @@ def test_NMOS(library, test_ckt):
                 model='NMOS',
                 pins={'D': 'NET10', 'G': 'NET12', 'S': 'NET13'},
                 generator='MOS')
-        with pytest.raises(Exception):
-            inst = Instance(
-                name='X1',
-                model='NMOS',
-                pins={'D': 'NET10', 'G': 'NET12', 'S': 'NET13', 'B': 'VSS'},
-                generator='MOS')
```

Comment:
See prior comment on prefix checks. Also Kunal's comment on capitalization.

Reply:
How can a PDK overwrite the default Library? Is there a way to do so?